src/main/java/com/ase/restservice/service/AccountService.java

```diff
@@ -0,0 +1,43 @@
+package com.ase.restservice.service;
+
+import com.ase.restservice.exception.ResourceNotFoundException;
+import com.ase.restservice.model.Account;
+import com.ase.restservice.repository.AccountRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+/**
+ * Service for Account operations.
+ */
+@Service
+public class AccountService {
+
+  @Autowired
+  private AccountRepository accountRepository;
+
+  /**
+   * Saves an account to the database.
+   *
+   * @param account Account
+   * @return Updated account
+   */
+  public Account save(Account account) {
```

Comment:
Could be a more descriptive name? like createNewAccount

Reply:
Changed this after implementing your other comment. Let's have separate create, update, delete methods for the services. I also created an interface to enforce this